### PR TITLE
Issue 126 redirect with oauth status

### PR DESCRIPTION
**Pull request by djlmills**
This PR has lots of rough edges but is sufficient to get the Github oAuth loop working.

### CODE REVIEW

File: packages/app-account/src/containers/Account.js [hunk 1]
```diff
@@ -54,20 +52,19 @@ const mapStateToProps = (state, ownProps) => {
     isAttentionMoreInfoModalVisible
   }
 
+  if (!selectors.isChainLoaded(state, { chainAlias: PRIVATE })) {
+    return notAuthenticating
+  }
+
   const {
```
Comment (djlmills) [inline packages/app-account/src/containers/Account.js]: Handling the redirect parameters will be pulled out into a separate component in a separate PR.

File: packages/app-account/src/containers/Account.js [hunk 2]
```diff
@@ -54,20 +52,19 @@ const mapStateToProps = (state, ownProps) => {
...
+  const { requestId, providerChainId, joinName } = query
   const { oAuthProvider } = params
 
-  if (!(oAuthProvider && code && requestId)) {
-    return notAuthenticating
-  }
-
-  const consumerChainId = selectors.getChainId(state.interbit, PRIVATE)
-  if (!consumerChainId) {
+  if (!(oAuthProvider && requestId && providerChainId && joinName)) {
```
Comment (nicholeous) [inline packages/app-account/src/containers/Account.js]: What is this checking for? Can this be a named variable?
